### DISCUSSION TITLE
[Bug]: exec /usr/local/bin/docker-entrypoint.sh: exec format error

### DISCUSSION THREAD

**Discussion by themw123** [danny-avila/LibreChat#1904] [Troubleshooting]
### What happened?

On Raspberry Pi 4 from docker logs i am getting exec `/usr/local/bin/docker-entrypoint.sh: exec format error` after `docker compose up -d`.

### Steps to Reproduce

fetch git repo
`docker compose up -d`
`docker logs LibreChat`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
it's working but recently the default build method was updated to use the pre-built images instead of building locally, so you may need to configure a docker compose override file to build the image locally. ~~The pre-built images may not be compatible for you for some reason (which is odd because we build both amd64 and arm64).~~

Did you ever configure to use the pre-built docker images before? https://github.com/danny-avila?tab=packages&repo_name=LibreChat

see this guide for more info:
https://docs.librechat.ai/install/configuration/docker_override.html

**Reply by danny-avila** [reply]
Actually to clarify, the dev images may not build images compatible with your system, in which case you should build locally or use the non-dev images: https://github.com/danny-avila?tab=packages&repo_name=LibreChat

**Reply by mhogstedt** [reply]
This solved the issue for me. 👍

**Reply by danny-avila** [reply]
awesome, thank you for confirming @mhogstedt 

I've now successfully built arm compatible builds for the dev images: 
https://github.com/danny-avila/LibreChat/pkgs/container/librechat-dev
